Dockerfile

```diff
@@ -1,9 +1,12 @@
-FROM alpine:3.14
+ARG ALPINE_VERSION=3.14
```

Comment:
Why did you make this a build arg?

Reply:
So that you can build an image with some specific version without changing the Dockerfile.

Reply:
Good one, I like that :+1: